omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -237,7 +245,7 @@
 
 # Paths to the SSL certificate and key files. If these are not provided we will
 # attempt to generate a self-signed certificate and use that instead.
-default['supermarket']['ssl']['enabled'] = true
+default['supermarket']['ssl']['enable'] = true
```

Comment:
Is there a reason to change this? This is going to be a really annoying breaking change if anyone set this.

Reply:
@tas50 Currently there are multiple flags through which we handle which service is supposed to be enabled e.g. `enable`, `enabled`. SSL is the only service that currently uses `enabled`. We should use a common standard for a single purpose. Also this is a needed change because we have refactored certain code segments to make things generalized and easy to manage through this `enable` flag. We should get rid of multiple formats for a single flag. Hence this change.

Ideally we should use some global variable which will represent this flag e.g. below:
`$ENABLED = "enable"`
and use the variable instead of string literal so that the value can be in a single place.